src/client.ts

```diff
@@ -54,14 +56,23 @@ export interface JobOptions {
 export class Client {
   /** Internal transport used to send requests to Zeus. */
   private readonly transport: Transport;
+  /** GitHub repository owner */
+  public readonly owner: string;
+  /** GitHub repository name */
+  public readonly repo: string;
```

Comment:
I'd prefer to not have `owner` / `repo` as part of the client configuration, but rather options in the methods.

Reply:
You're right: this is an sdk lib, so better to make things explicit.